### ISSUE TITLE
Documentation on --size argument introduced with #286

### ISSUE THREAD

**Issue by foxsan48** [sharkdp/fd#369]
The only way I worked out how to use `--size` was by reading the tests #f01685c63f05d842fc40afd98e4d3b1d1d5d534c
 perhaps document that in the help page and README.MD on here?

**Comment by sharkdp** [comment]
Both the man page as well as the `--help` text include this part:
```
    -S, --size <size>...                  
            Limit results based on the size of files using the format <+-><NUM><UNIT>.
               '+': file size must be greater than or equal to this
               '-': file size must be less than or equal to this
               'NUM':  The numeric size (e.g. 500)
               'UNIT': The units for NUM. They are not case-sensitive.
            Allowed unit values:
               'b':  bytes
               'k':  kilobytes
               'm':  megabytes
               'g':  gigabytes
               't':  terabytes
               'ki': kibibytes
               'mi': mebibytes
               'gi': gibibytes
               'ti': tebibytes
```

I don't really know what else you expected?

**Comment by foxsan48** [comment]
OH `-h` doesn't mention `--help` is more verbose so I thought they were the same, sorry

**Comment by sharkdp** [comment]
Agreed, it would be great if the `-h` included a short passage at the end that would refer to the more detailed `--help` text.

**Comment by sharkdp** [comment]
Closed in 985fab16bbe50d35216531b9f7476a3f33e55522
